image/gpt/ab_userdata/post-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eu
+
+upd="${1}/update"
+rm -rf "${1}/update"
+mkdir -p $upd
+
+[[ -f ${1}/system.sparse ]] || false
+[[ -f ${1}/boot.sparse ]] || false
+
+ln -sf ../system.sparse ${upd}/system
+ln -sf ../boot.sparse ${upd}/boot
+
+msg "Packing..."
+tar -I zstd -h -cf  ${1}/update.tar.zst  -C $upd .
```

Comment:
This looks neater than the version I was using, but it has the unfortunate side effect of generating entries in the tar file beginning `./`, which causes problems because the name doesn't match that of one of the A/B slots. Could you use this instead?
```
(cd $upd; tar -I zstd -h -cf  ${1}/update.tar.zst *)
```